### DISCUSSION TITLE
Increase contrast on input field

### DISCUSSION THREAD

**Discussion by olimart** [AllYourBot/hostedgpt#199] [General]
Current contrast is too low when text is entered.
(placeholder is fine)

Here is an example (first field has text, second field only placeholder)
<img width="397" alt="Capture d’écran, le 2024-03-16 à 00 32 22" src="https://github.com/the-dot-bot/hostedgpt/assets/547754/9df01a8e-85c7-4a44-9cf1-e06c61b96874">

**Comment by keithschacht** [comment]
@olimart Ah, this is a dark mode issue. Thanks for flagging this. I merged in a fix for this and I audited all the other form fields and found a few others I had missed.
